modular_skyrat/modules/modular_reagents/code/debuffs.dm

```diff
@@ -0,0 +1,51 @@
+/datum/status_effect/grouped/stasis_mobile
+	id = "stasis"
+	duration = -1
+	tick_interval = 10
+	alert_type = /atom/movable/screen/alert/status_effect/stasis
+	var/last_dead_time
+
+/datum/status_effect/grouped/stasis_mobile/proc/update_time_of_death()
```

Comment:
documentation please

Reply:
Documentation for a proc should at the minimum say what the proc does, and usually should say what each parameter represents/is used for, and finally what the proc returns, if it returns anything. I'm not on my computer so I can't give you a great example here, but for an example of what's the kind of documentation I *really* like, look at `conveyor.dm` (or maybe it's called `recycling.dm`, I don't remember the name).